src/main/java/com/google/devtools/build/lib/rules/java/DeployArchiveBuilder.java

```diff
@@ -258,7 +258,7 @@ public void build() throws InterruptedException {
         outputJar, javaStartClass, deployManifestLines, buildInfoArtifacts, classpathResources,
         runtimeClasspath, includeBuildData, compression, launcher);
 
-    List<String> jvmArgs = ImmutableList.of("-client", SINGLEJAR_MAX_MEMORY);
```

Comment:
/cc @philwo: why do we still have -client flag on Java on some machine, is there case it would be needed?